docs/modules/ROOT/pages/reference/extensions/xmlsecurity.adoc

```diff
@@ -33,3 +33,14 @@ Please refer to the above links for usage and configuration details.
 ----
 
 Check the xref:user-guide/index.adoc[User guide] for more information about writing Camel Quarkus applications.
+
+== Camel Quarkus limitations
+
+There is currently no native mode support for XSLT based transform methods on the `xmlsecurity` producer via the `transformMethods` URI option.
+
```

Comment:
Would it be interesting to describe the error message the user will see ? Or maybe in the related ticket ?

Reply:
I didn't really want to go into the specifics of the problems in the README. But the native build will fail since there is some missing reflective class config + some other bits. I described the problem (very) briefly in https://github.com/apache/camel-quarkus/issues/2185.

Reply:
fair enough